File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/FillTextCommand_ViewModel.cpp
#include "FillTextCommand_ViewModel.h"
#include <vector>
#include <string>

namespace commands
{
  std::string FillTextCommand_ViewModel::getMyValueTextBoxText()
  {
    return this->MyValueTextBoxText;
  }
  std::vector<std::string> FillTextCommand_ViewModel::getMyFreeValueComboBoxEntries()
  {
    return this->MyFreeValueComboBoxEntries;
  }
  std::string FillTextCommand_ViewModel::getMyFreeValueComboBoxSelectedEntry()
  {
    return this->MyFreeValueComboBoxSelectedEntry;
  }
  std::string FillTextCommand_ViewModel::getMyFreeValueComboBoxText()
  {
    return this->MyFreeValueComboBoxText;
  }
  void FillTextCommand_ViewModel::myValueTextBoxTextFilled(std::string text)
  {
  }
  void FillTextCommand_ViewModel::myFreeValueComboBoxTextFilled(std::string text)
  {
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/FillTextCommand_ViewModel.cs
/// <filename>
///     FillTextCommand_ViewModel.cs
/// </filename>
using System.Collections.Generic;

namespace commands
{
    public class FillTextCommand_ViewModel
    {
        private string MyValueTextBoxText;

        public virtual string getMyValueTextBoxText()
        {
            return this.MyValueTextBoxText;
        }

        private System.Collections.Generic.List<string> MyFreeValueComboBoxEntries = new System.Collections.Generic.List<string>();

        public virtual System.Collections.Generic.List<string> getMyFreeValueComboBoxEntries()
        {
            return this.MyFreeValueComboBoxEntries;
        }

        private string MyFreeValueComboBoxSelectedEntry;

        public virtual string getMyFreeValueComboBoxSelectedEntry()
        {
            return this.MyFreeValueComboBoxSelectedEntry;
        }

        private string MyFreeValueComboBoxText;

        public virtual string getMyFreeValueComboBoxText()
        {
            return this.MyFreeValueComboBoxText;
        }

        public virtual void myValueTextBoxTextFilled(string text)
        {
        }

        public virtual void myFreeValueComboBoxTextFilled(string text)
        {
        }
    }
}
